doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc

```diff
@@ -60,32 +60,39 @@ NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set
-* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in {CENTRAL} `.security` folder.
-* `appformer.experimental.features`: Enables the Experimental Features Framework.
-* `org.kie.demo`: Enables external clone of a demo application from GitHub.
-* `org.kie.workbench.controller`: The URL is used to connect with the {CONTROLLER}. For example, `ws://localhost:8080/kie-server-controller/websocket/controller`.
-* `org.kie.workbench.controller.user`: The user name is used to connect with the {CONTROLLER}. Default value: kieserver.
-* `org.kie.workbench.controller.pwd`: The password is used to connect with the {CONTROLLER}. Default value: kieserver1!.
-* `org.kie.workbench.controller.token`: The token string is used to connect with the {CONTROLLER}.
+* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in the {CENTRAL} `.security` folder.
+COMMENT: Default value?
+* `appformer.experimental.features`: Enables the experimental features framework.
+COMMENT: Default value?
+* `org.kie.demo`: Enables an external clone of a demo application from GitHub.
+COMMENT: Default value?
+* `org.kie.workbench.controller`: The URL used to connect to the {CONTROLLER}, for example, `ws://localhost:8080/kie-server-controller/websocket/controller`.
+COMMENT: Default value?
+* `org.kie.workbench.controller.user`: The {CONTROLLER} user. Default value: `kieserver`.
+* `org.kie.workbench.controller.pwd`: The {CONTROLLER} password. Default value: `kieserver1!`.
+* `org.kie.workbench.controller.token`: The token string used to connect to the {CONTROLLER}.
 +
 ifdef::DROOLS,JBPM[]
 NOTE: For more information about how to use token based authentication, see xref:usingTokenBasedAuthentication[].
 endif::[]
 
-* `kie.keystore.keyStoreURL`: The URL is used to load a Java Cryptography Extension KeyStore (JCEKS). For example, `\file:///home/kie/keystores/keystore.jceks.`
-* `kie.keystore.keyStorePwd`: The password is used for the JCEKS.
-* `kie.keystore.key.ctrl.alias`: The alias of the key for default REST {CONTROLLER}.
-* `kie.keystore.key.ctrl.pwd`: The password of an alias for default REST {CONTROLLER}.
-* `org.jbpm.wb.forms.renderer.ext`: Switches the form rendering between {CENTRAL} and {KIE_SERVER}. By default, the form rendering is done by {CENTRAL}. Default value: `false`.
-* `org.jbpm.wb.forms.renderer.name`: This property allows to switch between {CENTRAL} and {KIE_SERVER} rendered forms. Default value: `workbench`.
+* `kie.keystore.keyStoreURL`: The URL used to load a Java Cryptography Extension KeyStore (JCEKS). For example, `\file:///home/kie/keystores/keystore.jceks.`
```

Comment:
`kie.keystore.keyStoreURL` - default value is `""`

https://github.com/kiegroup/kogito-runtimes/blob/master/drools/drools-core/src/main/java/org/drools/core/util/KeyStoreConstants.java#L32

https://github.com/kiegroup/drools/blob/master/drools-core/src/main/java/org/drools/core/util/KeyStoreHelper.java#L118